Contribute/Style-Guide/index.md

```diff
@@ -0,0 +1,81 @@
+# Style Guide
+
+To ensure that the documentation is readable and has a similar style througout we have a few rules to follow. These rules are set up with an automatic style linter called [Vale](https://errata-ai.github.io/vale/). Additionally we have a Github bot that will check PRs for broken rules and advise you what to change.
+
+## Current style rules
+
+### Punctuation in headings
+
+This rule will give a warning if you end headings with either of these punctuation symbols:
+
+- ?
+- :
+- .
+
+### Use of Hyperbole
+
+This rule will give a warning if you use either of these words in your text:
+
+- simple
+- just
+- easily
+- actually
+
+### Start lists with capital letters
+
+This rule will give an error if you have a list that starts with uncapitalized words.
+
+### Long sentences
+
+This rule will give an error if you have a sentence with more than 40 words. The number of words may change in the future, generally 20-25 is the recommended amount per sentence in documentation.
+This is to improve readability.
+
+### Double spacing
+
+This rule will warn you if you have more than 1 space in a row in your text.
+
+### Terms
+
+This rule will suggest a different term if you use one we have deemed wrong, this is to ensure the same spelling of terms throughout the documentation.
+
+**Example**:
+The term `backoffice` has been seen spelled in many different ways, most commonly:
+
+1. `backoffice`
+1. `back office`
+1. `back-office`
+
+This rule will kick in if you spell it as number 2 or 3 and suggest you spell it as number 1. For a full list of terms please check the [style rule](https://github.com/umbraco/UmbracoDocs/tree/master/.github/vale/DocsStyles/Terms.yml).
+
+## Using Vale locally
+
+One of the big strengths of Vale is that it is possible as a contributor to simply run the tests locally before you create a PR, there is also a VS Code extension for it.
+
+First step to running Vale locally is to install it following [their documentation](https://errata-ai.github.io/vale/#installation).
+
+Next you can open a command line tool in the documentation repo and run the following command:
+
+```vale --glob='*.md' .```
```

Comment:
Not sure what's wrong here, but it looks like the formatting of the rest of the article is a bit funky after this line 🤔

Reply:
It looked fine in a Markdown previwer locally, and also looks fine if you preview the document in Github, but yea the syntax coloring is weird in Github after this..

Reply:
Probably just a GitHub thing then 😁

Reply:
Maybe try replacing it with 4 spaces in front.
Or put in on three lines.